### PR TITLE
Feature/add tcp option

### PR DESCRIPTION
**Pull request by StephenKinger**
I have add a new menu for tcp option instead of udp which can be useful for users who needs to use 443 port for example in a company to connect to an external vpn.

### CODE REVIEW

File: auto_install/install.sh [hunk 1]
```diff
@@ -475,6 +475,32 @@ update_repo() {
     echo " done!"
 }
 
+setCustomProto() {
```
Comment (StephenKinger) [inline auto_install/install.sh]: yes, we can add a message to say that UDP is recommanded

Comment (redfast00) [inline auto_install/install.sh]: @StephenKinger @0-kaladin: added the message and some cleanup. Should be good to go

**Comment by StephenKinger** [comment]
Ok, i checked and it's work. I can set up automatically the 443 port in case of tcp is choosen (useful for vpn setup used to be accessed from filtered network) if you want.

**Comment by 0-kaladin** [comment]
Looks good guys, great work!

**Pull request merged**